packages/create-astro/src/messages.ts

```diff
@@ -25,77 +25,84 @@ export const welcome = [
     `Awaiting further instructions.`,
 ]
 
-export const getName = () => new Promise((resolve) => {
-  exec('git config user.name', { encoding: 'utf-8' }, (_1, gitName, _2) => {
-    if (gitName.trim()) {
-        return resolve(gitName.split(' ')[0].trim());
-    }
-    exec('whoami', { encoding: 'utf-8' }, (_3, whoami, _4) => {
-        if (whoami.trim()) {
-            return resolve(whoami.split(' ')[0].trim());
-        }
-        return resolve('astronaut');
-    });
-  });
-});
+export function getName() {
```

Comment:
Ah this is cool,

Reply:
this is so cool but... is it creepy? I guess we can launch it and find out!

Reply:
I feel exactly the same way. We'll see what people say, I'm not opposed to dropping this if anyone finds it weird.